versioned_docs/version-8.1/reference/supported-environments.md

```diff
@@ -34,7 +34,7 @@ Requirements for the components can be seen below:
 
 | Component                | Java version | Other requirements                                                                               |
 | ------------------------ | ------------ | ------------------------------------------------------------------------------------------------ |
-| Zeebe Broker and Gateway | OpenJDK 17+  | Elasticsearch 7.16.x (only if Elastic exporter is used)                                          |
+| Zeebe Broker and Gateway | OpenJDK 17+  | Elasticsearch 7.16.x, 7.17.x (only if Elastic exporter is used)                                  |
```

Comment:
```suggestion
| Zeebe Broker and Gateway | OpenJDK 17+  | Elasticsearch 7.16.x, 7.17.x (only if Elastic exporter is used)                                  |
```

I know this already said Zeebe Broker and Gateway, but this is a bit strange. The gateway has nothing to do with Elasicsearch at all. Adding broker seems lik too much of an implementation detail to me. It's better to just refer to the component as `Zeebe`.